src/hooks/v2/transactor/ClaimTokensTx.ts

```diff
@@ -15,6 +15,22 @@ export function useClaimTokensTx(): TransactorInstance<{
 
   return ({ claimAmount }, txOpts) => {
     if (!transactor || !userAddress || !projectId || !contracts?.JBTokenStore) {
+      const missingParam = !transactor
+        ? 'transactor'
+        : !userAddress
+        ? 'userAddress'
+        : !projectId
+        ? 'projectId'
+        : !contracts?.JBTokenStore
+        ? 'contracts.JBTokenStore'
+        : null
+
+      txOpts?.onError?.(
+        new DOMException(
+          `Missing ${missingParam ?? 'parameter` not found'} in v2 transactor`,
```

Comment:
I don't think we need 'parameter`' --> 'parameter' remove the `

Reply:
nice catch!